### ISSUE TITLE
KeyError: 'delete_selected'

### ISSUE THREAD

**Issue by diegueus9** [indietyp/django-automated-logging#2]
I was testing this app with django 2.1.8 and got this error when trying to check the admin:

```python
File "/var/virtualenv/project/lib/python3.6/site-packages/automated_logging/admin.py", line 19, in get_actions
    del actions["delete_selected"]
KeyError: 'delete_selected'
```

An easy fix would be:
```python
    def get_actions(self, request):
        actions = super(ReadOnlyAdminMixin, self).get_actions(request)
        actions.pop('delete_selected', None)
        return actions
```

**Comment by indietyp** [comment]
I am sorry for the late reply - I haven't been able to keep track of the project as much as I would've wanted in the last year.

Issue is acknowledged and will be fixed in a future release (currently working on a rework)

**Comment by indietyp** [comment]
fixed with 6.0.0
